packages/upload-api/src/provider-add.js

```diff
@@ -29,6 +31,22 @@ export const add = async (
       },
     }
   }
+  const isBlocked = await areAnyBlocked(
+    rateLimits,
+    emailAndDomainFromMailtoDid(
+      /** @type {import('@web3-storage/did-mailto/dist/src/types').DidMailto} */ (
+        accountDID
+      )
+    )
+  )
+  if (isBlocked.error || isBlocked.ok) {
+    return {
+      error: {
+        name: 'AccountBlocked',
+        message: `Account identified by {capability.nb.iss} is blocked`,
```

Comment:
I would expect to report accountDID instead since that is what is checked above. Also there `$` was missing in template literal.

```suggestion
        message: `Account identified by ${accountDID} is blocked`,
```